src/test/ast_program_test.cpp

```diff
@@ -24,6 +24,21 @@ namespace souffle {
 
 namespace test {
 
+inline std::unique_ptr<AstTranslationUnit> makeATU(std::string program = ".decl A,B,C(x:number)") {
+    SymbolTable sym;
+    ErrorReport e;
+    DebugReport d;
+    return ParserDriver::parseTranslationUnit(program, sym, e, d);
+}
+
+inline std::unique_ptr<AstClause> makeClauseA(std::unique_ptr<AstArgument> headArgument) {
```

Comment:
I think it might be better to just call this `makeClause` and pass in the clause name as the first argument (`std::string`). That way, it'll be a lot more versatile for future tests we might add.